lightly_studio/tests/api/routes/api/test_text_embedding.py

```diff
@@ -11,11 +12,14 @@
     EmbeddingManager,
     EmbeddingManagerProvider,
 )
+from tests import helpers_resolvers
 
 
-def test_embed_text(mocker: MockerFixture, test_client: TestClient) -> None:
+def test_embed_text(db_session: Session, mocker: MockerFixture, test_client: TestClient) -> None:
     # Initialize the embedding_manager with a mock variant so it does not update
     # the singleton.
```

Comment:
Minor: This comment is relevant to the `mocker.patch.object` line, should be a bit lower.